### DISCUSSION TITLE
Tiptap integration?

### DISCUSSION THREAD

**Discussion by davidpene** [stepwisehq/prosemirror-dotnet#6] [Q&A] [answered]
Can this be used to integrate with Tiptap?
The frontend could send [deltas (transactions)](https://tiptap.dev/api/events#transaction) to a backend which would handle the reconciliation process.

**Comment by ProTip** [comment, marked as answer]
Hi @davidpene ,

Yes as TipTap is "just" an API wrapper built on top of Prosemirror. We are currently using this project on the backend along with our [Collab Commit Plugin](https://github.com/stepwisehq/prosemirror-collab-commit); written about [here](https://stepwisehq.com/blog/2023-07-25-prosemirror-collab-performance/). On the front-end we are currently using TipTap.

We don't have a full example of this ATM, but the `Prosemirror.Net` companion APIs for Collab Commit are here: https://github.com/stepwisehq/prosemirror-dotnet/tree/main/src/Collab .
